src/set_theory/ordinal_arithmetic.lean

```diff
@@ -1877,6 +1919,9 @@ theorem add_mul_limit {a b c : ordinal} (ba : b + a = a)
   (l : is_limit c) : (a + b) * c = a * c :=
 add_mul_limit_aux ba l (λ c' _, add_mul_succ c' ba)
 
+theorem add_mul_omega (a) : a + a * omega = a * omega :=
```

Comment:
This seems like an overly specific lemma. It is just a slight variant on `1 + omega = omega`, can we do without it?

Reply:
Fair point, the lemma has been removed.